src/pages/docs/service-level-agreement.md

```diff
@@ -0,0 +1,111 @@
+
+## 3. Service Discount Claims.
+
+3.1 Company provides this SLA subject to the following terms.
+
+3.2 In order to be eligible to submit a Claim with respect to any Incident, the Customer must first have notified Customer Support of the Incident, using the procedures set forth by Company in Section 3.3, within five business days following the Incident.
+
+3.3 To submit a Claim, Customer must contact Customer Support at support@web3.storage and provide notice of its intention to submit a Claim. Customer must provide to Customer Support all reasonable details regarding the Claim, including but not limited to, detailed descriptions of the Incident(s), the duration of the Incident, network traceroutes, the Content ID(s) or file(s) affected and any attempts made by Customer to resolve the Incident.
+
+3.4 In order for Company to consider a Claim, Customer must submit the Claim, including sufficient evidence to support the Claim, by the end of the billing month following the billing month in which the Incident which is the subject of the Claim occurs.
+
+3.5 Company will use all information reasonably available to it to validate Claims and make a good faith judgment on whether the SLA and Service Levels apply to the Claim.
+
+## 4. SLA Exclusions.
+
+4.1 This SLA and any applicable Service Levels do not apply to any storage, performance or availability issues:
+
+(a) Due to factors outside Company’s reasonable control;
+
+(b) That resulted from Customer’s or third party hardware or software;
+
+(c) That resulted from actions or inactions of Customer or third parties;
+
+(d) Caused by Customer’s use of the Services after Company advised Customer to modify its use of the Services, if Customer did not modify its use as advised;
+
+(e) During beta and trial Services (as determined by Company);
+
+(f) Attributable to the acts or omissions of Customer or Customer’s employees, agents, contractors, or vendors, or anyone gaining access to Company’s Services by means of Customer’s Authorized Users’ accounts or equipment;
+
+(g) Caused by Customer attempting to upload incompatible data types or raw files larger than 100 MB via the Upload Service;
+
+(i) Caused by Customer attempting to upload files via the web3.storage website;
+
+(j) Caused by Customer attempting to store files via the pinning service API;
+
+(k) Due to failures of infrastructure used to read data from web3.storage, including inaccessibility of public gateways;
+
+(l)  That resulted from Customer’s use of gateways not peered with web3.storage (i.e., not listed on <https://github.com/web3-storage/web3.storage/blob/main/PEERS>);
```

Comment:
What to do with the peers file? Migrate to w3up?

Reply:
will talk with alan